From 5fcca4b9657ee5e64664f8fe1fdd7288b9704858 Mon Sep 17 00:00:00 2001
From: Jiawei Huang <jiawei@tigera.io>
Date: Wed, 4 Jun 2025 19:25:06 -0700
Subject: [PATCH] Rename Fluent Bit package name to Calico Fluent Bit

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bc5990edb..5d70cd6e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1039,8 +1039,8 @@ else()
 
 endif()
 
-set(FLB_PROG_NAME "Fluent Bit")
-set(FLB_OUT_NAME "fluent-bit")
+set(FLB_PROG_NAME "Calico Fluent Bit")
+set(FLB_OUT_NAME "calico-fluent-bit")
 
 if(FLB_PROXY_GO)
   FLB_DEFINITION(FLB_HAVE_PROXY_GO)
-- 
2.49.0

