From 1c66dc36e1017934d4e262a38fe1872f32a031de Mon Sep 17 00:00:00 2001
From: Jiawei Huang <jiawei@tigera.io>
Date: Wed, 4 Jun 2025 19:25:26 -0700
Subject: [PATCH] Move config files to /etc/calico/calico-fluent-bit

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5d70cd6e3..74f81d3ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1353,7 +1353,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 else()
   set(FLB_INSTALL_BINDIR ${CMAKE_INSTALL_FULL_BINDIR})
   set(FLB_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/${FLB_OUT_NAME}")
-  set(FLB_INSTALL_CONFDIR "${CMAKE_INSTALL_SYSCONFDIR}/${FLB_OUT_NAME}/")
+  set(FLB_INSTALL_CONFDIR "${CMAKE_INSTALL_SYSCONFDIR}/calico/${FLB_OUT_NAME}/")
   set(FLB_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include")
 endif()
 
-- 
2.49.0

